src/main/java/com/ita/if103java/ims/dao/impl/AccountTypeDaoImpl.java

```diff
@@ -170,5 +179,7 @@ private PreparedStatement createStatement(AccountType accountType, Connection co
         static final String SQL_SET_ACTIVE_STATUS_ACCOUNT_TYPE = "UPDATE account_types SET active = ? WHERE id = ?";
 
         static final String SQL_FIND_MIN_LVL_TYPE = "SELECT MIN(level) FROM account_types";
+
+        static final String SQL_FIND_ALL_POSSIBLE_TO_UPGRADE = "SELECT * FROM account_types WHERE level > ?";
```

Comment:
I was thinking about this and decided that we need to select all. Please left this method for now, but most probably we will not use it and remove in future